typography/typography.css

```diff
@@ -0,0 +1,67 @@
+/* Section container for typography items */
+.disp-typo {
+  display: flex;
+  padding: 0 20rem;
+  flex-direction: column;
+  text-align: left;
+  margin: 1rem;
+}
+
+.sub-title {
+  font-size: 1.2rem;
+}
+
+/* Changed 48px & 64px into rems */
+.h-1 {
+  font-size: 3rem;
+  line-height: 4rem;
+}
+
+/* Changed 32px & 40px into rems */
+.h-2 {
+  font-size: 2rem;
+  line-height: 2.5rem;
+}
+
+/* Changed 24px & 32px into rems */
+.h-3 {
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+/* Changed 16px & 24px into rems */
+.h-4 {
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+/* Changed 8px & 16px into rems */
+.h-5 {
+  font-size: 0.5rem;
+  line-height: 1rem;
+}
+
+.lg {
+  font-weight: 200;
+}
+
+.rg {
+  font-weight: 400;
+}
+
+.sm-bold {
+  font-weight: 600;
+}
+.bold {
+  font-weight: 700;
+}
+.black {
```

Comment:
extra-bold would be a better class name than black IG

Reply:
Yaa, I'll update it. Thanks for your valuable suggestions.